src/components/i18n/en.tsx

```diff
@@ -44,6 +44,10 @@ will not be applied.`,
   new_template_title: `Save as private Template`,
   products_gallery_empty_title: `There are no products in the store.`,
   products_gallery_search_no_results_message: `There's no product with the name "{searchTerm}"`,
+  products_gallery_view_default_price: `Price`,
```

Comment:
Yo haria referencia a que son los textos de los headers (pero como vos prefieras)
```suggestion
  products_gallery_view_default_price_header: `Price`,
```